firebase.json

```diff
@@ -47,6 +47,7 @@
       "port": 9199,
       "host": "127.0.0.1"
     }
+    "singleProjectMode": true
```

Comment:
Your changes are still present in `firebase.json`. Reset the commit, then compare the changes, and remove the changes in `firebase.json`